README.md

```diff
@@ -65,6 +65,19 @@ Style the component the way you want (example):
 +}
 ```
 
+**Note**:  
+If you want to style `ngx-row-accordion` globally, put those styles into `src/styles.scss`.
+
+If you want to style accordions inside a given component, you should use `::ng-deep` like the following:
+
+```diff
++::ng-deep ngx-row-accordion {
```

Comment:
It should be the other way around i.e  `ngx-row-accordion ::ng-deep {}`

Reply:
Thanks, updated!